hooks/post/30-upgrade.rb

```diff
@@ -66,6 +66,15 @@ def remove_gutterball
   end
 end
 
+def remove_event_queue
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
```

Comment:
@ehelms @ekohl I do know @jlsherrill is working on a PR that will not run already completed steps when the installer runs with --upgrade so this would only get hit once and really by downstream 6.3 most customers should not have this issue/queue anymore. I can always revisit this if needed if the path does change. Let me know how to proceed?